src/StreamingDataFrames/requirements.txt

```diff
@@ -0,0 +1,2 @@
+confluent-kafka>=2.2,<2.3
+requests>=2.31,<2.4
```

Comment:
I think I'm the one who did this...but I think this should be <2.32?

Reply:
I think it's fine to keep it this way for now.

I was thinking about relaxing the `requests` version even more in case `quixstreams` is used as a library and `requests` is already in the project's requirements, so we don't enforce the version too much